application/templates/error/400.html

```diff
@@ -9,5 +9,8 @@
             <div class='column-full'>
                 <h1 class="heading-large">Bad Request</h1>
             </div>
+            <div class='column-two-thirds'>
```

Comment:
We probably only want to do this if there is a message. By default it's an empty string, so this might break it with a random div in there?

Reply:
:+1: